README.md

```diff
@@ -18,21 +18,59 @@ These the Airflow expectations for these Funnel Cake DAGs to successfully run:
 
 **Libraries & Packages**
 
-- Python 3.7
+- Python: Version as specified in `.python-version`, currently 3.6.8.
 - Python Packages: see the [Pipfile](Pipfile)
 
 **Airflow Variables**
 
-See `variables.json` file
+These variable are initially set in `variablesjson`
+
+- `AIRFLOW_DATA_BUCKET`: The AWS S3 Bucket label (label / name, not ARN or URI) the harvested OAI-PMH XML data is put into / indexed from.
+- `MANIFOLD_BLOGS_SYNC_SCHEDULE_INTERVAL`: Cron schedule to sync blogs. (Usually `@daily`)
+- `MANIFOLD_DUMP_DATABASE_SCHEDULE_INTERVAL`: Chron schedule to dump the Manifold database to AWS (Usually `@weekly`)
+- `MANIFOLD_EVENTS_SYNC_SCHEDULE_INTERVAL`: Cron schedule to syncrhonize library events (Usually `@daily`)
+- `MANIFOLD_HOURS_SYNC_SCHEDULE_INTERVAL`: Cron schedule to syncronize library hours (Usually `@daily`)
+- `TUP_ACCOUNT_NAME`: Account used to access TUPress to retrieve blog RSS feed 
+- `TUPRESS_HARVEST_SCHEDULE_INTERVAL`: Cron schedule to syncrhoize TUPress blog feed (Usually 8AM daily - `0 8 * * *`)
+- `TUPRESS_SFTP_PATH`: destination server file path for delta file (where you will put the tupress delta file for processing)
+- `TUPRESS_WEB_PATH`: source server File path to delta file on the TUPress web server (Where you will get the tupress delta file for processing)
+
+See the rest of the expected variables in `variables.json`.
 
 **Airflow Connections**
 - `SOLRCLOUD`: An HTTP Connection used to connect to SolrCloud.
 - `AIRFLOW_S3`: An AWS (not S3 with latest Airflow upgrade) Connection used to manage AWS credentials (which we use to interact with our Airflow Data S3 Bucket).
 
 ## Local Development
 
+### Run with Airflow Playbook Make Commands
```

Comment:
@nomadicoder I'm not sure these are the best directions.  We use airflow-docker-dev-setup as a submodule and that should be what people are using for local development.  Since we are planning to add additional team members at some point, I think it would be better for the instructions to follow the current setup.

The README for using airflow-docker-dev-setup is pretty thorough.  Maybe you could explain the connection and provide a link to that README for usage. instructions.

Reply:
Should we have them `cd airflow-docker-dev-setup` and start up containers from there. If that's the case, then I should reorganize the documentation to have the user configure the application DAG.

there is one sticking point here, also, and that is if you follow the original directions, then the application DAG (i.e. `manifold_airflow_dag`) installs the airflow-docker-dev-setup with `git submodule init` and `git submodule update`  I think the application DAG `README.md` should instruct the user to perform these two steps and set up the application DAG config variables, then cd into `airflow-dag-dev-setup`, adjust the sleep time if necessary, and perform the `make up` from there.

Reply:
@nomadicoder That is my process for starting things locally. I cd in to airflow-docker-dev and then run my make commands. I also do that inside of a Pipenv shell so that I am working in a virtual environment.  I'm not sure what you mean by sticking point.  Are you saying you don't think the submodule commands should be part of the Make command?